core/src/main/res/values-as/strings.xml

```diff
@@ -37,4 +32,5 @@
     <item>@string/help_18</item>
     <item>@string/help_19</item>
   </string-array>
+  <plurals name="books_count">
```

Comment:
not a valid `plurals`

Reply:
Indeed, and this is going to be a bit harder to manage.